src/hardware/TempSensorDallas.cpp

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file TempSensorDallas.cpp
+ * 
+ * @brief Handler for Dallas DS18B20 temperature sensor
+ */
+
+#include "TempSensorDallas.h"
+#include <DallasTemperature.h> 
+
+OneWire* oneWire;
+DallasTemperature* dallasSensor;
+DeviceAddress sensorDeviceAddress;
```

Comment:
If they are only used by this class I would move them into the class as private members. In case this needs to be shared between multiple `TempSensorDallas` instances (which is not the case here) you could then mark them as `static`.